### PR TITLE
Fix NLog orchard-tenant-name returns None.

### LINKED ISSUE #1972: NLog not working
There is no log file generated.  However relevant logs can be seen in console.

### PR DESCRIPTION
**Pull request by awyl**
Fixes #1972

### CODE REVIEW

File: src/OrchardCore/OrchardCore.Logging.NLog/TenantLayoutRenderer.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ protected override void DoAppend(StringBuilder builder, LogEventInfo logEvent)
             var context = HttpContextAccessor.HttpContext;
 
             // If there is no ShellSettings in the Features then the log is rendered from the Host
```
Comment (jtkech) [inline src/OrchardCore/OrchardCore.Logging.NLog/TenantLayoutRenderer.cs]: Just update the comment with `ShellContext` in place of `ShellSettings`.

**Comment by jtkech** [comment]
Okay cool, you just need to update the comment ;)

**Comment by jtkech** [comment]
Also, you need to write `Fixes` before the issue number so that it is closed automatically when the related pr is merged. I updated your comment.

**Comment by awyl** [comment]
Ah, thanks.  Didn't know I have to put  `Fixes` in front.  Learnt something.  =)

**Pull request merged**